centreon/cron/centreon-backup.pl

```diff
@@ -302,9 +302,9 @@ ()
     if (defined($MYSQL_CONF)) {
         if ( -e $MYSQL_CONF) {
             return $MYSQL_CONF;
-        }
-    } elsif ( -e '/etc/my.cnf' ) {
-        return '/etc/my.cnf';
+        } elsif ( -e '/etc/my.cnf' ) {
+            return '/etc/my.cnf';
+        }    
```

Comment:
I have a question here.
Before the patch, we worked on /etc/my.cnf only when $MYSQL_CONF is not defined.
Now, you propose to work on /etc/my.cnf only when MYSQL_CONF is defined.

Would it not be better to work with /etc/my.cnf in both cases?

Reply:
I don't know the context, maybe I'm talking a lot of nonsense

Reply:
I assume that if the user did not put anything in the option "MySQL configuration file path" on the Centreon UI then the my.cnf conf was not saved
By the way, this file doesn't contain a specific conf in 99% of the cases